<!DOCTYPE HTML>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Xu lab@CBD_SCS_CMU</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="">
<meta name="author" content="">
<!-- css -->
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<link href="css/style.css" rel="stylesheet">

</head>
<body>
<!-- navbar -->
<div class="navbar-wrapper">
	<div class="navbar navbar-inverse navbar-fixed-top">
		<div class="navbar-inner">
			<div class="container">
				<!-- Responsive navbar -->
				<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse"><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span>
				</a>
				<h1 class="brand"><a href="index.html" tppabs="#">Xu lab@CBD_SCS_CMU</a></h1>
				<!-- navigation -->
				<nav class="pull-right nav-collapse collapse">
				<ul id="menu-main" class="nav">
					<li><a id="2" href="#news">News</a></li>
					<li><a id="3" href="#papers">Papers</a></li>
					<li><a id="4" href="#group">Group</a></li>
					<li><a id="5" href="#aboutxu">About Min Xu</a></li>
                    <li><a id="6" href="#position">Position</a></li>
                    <li><a id="7" href="#software">Software</a></li>
				</ul>
				</nav>
			</div>
		</div>
	</div>
</div>

<div class="window">

    <div id="news_page">
        <p class="title">News</p>
        <p class="text">
            <strong>200224:</strong> Congratulations to Xiangrui Zeng for his first authored paper gets accepted by CVPR 2020, the best Computer Vision conference!
        </p>
        <p class="text">
            <strong>191218:</strong> Congratulations to Kai Wen Wang for Honorable Mention for the Computing Research Association's (CRA) Outstanding Undergraduate Researcher Award for 2020!
        </p>
        <p class="text">
            <strong>190519</strong>: Congratulations Yixiu Zhao for being awarded 2019 Elizabeth W. Jones Award for Excellence in Undergraduate Research in Experimental or Computational Biology.
        </p>
        <p class="text">
            <strong>190415</strong>: Congratulations Xiangrui Zeng for being awarded the CMLH fellowship!
        </p>
        <p class="text">
            <strong>181104</strong>: Congratulations Bo Zhou for having the following paper accepted by BIBM-MLHRM 2018: Feature Decomposition based Saliency Detection in Electron Cryo-Tomograms
        </p>
        <p class="text">
            <strong>180917</strong>: Congratulations Ran Li, Xiangrui Zeng and other students for having the following paper accepted by APBC2019: Automatic Localization and Identification of Mitochondria in Cellular Electron Cryo-Tomography using Faster-RCNN
        </p>
        <p class="text">
            <strong>180822</strong>: Our following paper got accepted by Structure. De novo structural pattern mining in cellular electron cryo-tomograms.
        </p>
        <p class="text">
            <strong>180702</strong>: Congratulations Chang Liu and other students for having following paper accepted: Liu C, Zeng X, Guo Q, Wang K, Xu M. Multi-task Learning for Macromolecule Classification, Segmentation and Coarse Structural Recovery in Cryo-Tomography. British Machine Vision Conference (BMVC) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.06332&sa=D&sntz=1&usg=AFQjCNEJAdsHh1ZV1DtigT9hzbjzi-DKfw" target="_blank">arXiv:1805.06332</a>
        </p>
        <p class="text">
            <strong>180702</strong>: Congratulations Kaiwen Wang and other students for having following paper accepted: Wang K, Zeng X, Liang X, Huo Z, Xing E, Xu M. Image-derived generative modeling of pseudo-macromolecular structures - towards statistical assessment of electron cryotomography template matching. British Machine Vision Conference (BMVC) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.04634&sa=D&sntz=1&usg=AFQjCNGKj3NyL-PIBryMQi3zOw1_EJZ6fw" target="_blank">arXiv:1805.04634</a>
        </p>
        <p class="text">
            <strong>180525</strong>: Congratulations to Guannan Zhao, Bo Zhou and other students for having following paper accepted by MICCAI, a prestigious conference in biomedical image analysis! Zhao G, Zhou B, Wang K, Jiang R, Xu M. Respond-CAM: Analyzing Deep Models for 3D Imaging Data by Visualizations. Medical Image Computing and Computer Assisted Intervention (MICCAI) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1806.00102&sa=D&sntz=1&usg=AFQjCNGgP4R3YTn18pWjf3tItO1EJcBocg" target="_blank">arXiv:1806.00102</a>
        </p>
        <p class="text">
            <strong>180518</strong>: Congratulations to Chengqian Che and other students for having following paper accepted: Che C, Lin R, Zeng X, Elmaaroufi K, Galeotti J, Xu M. Improved deep learning based macromolecules structure classification from electron cryo tomograms. Machine Vision and Applications. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1707.04885&sa=D&sntz=1&usg=AFQjCNFVQ0NVTvzjbm5EVYnmj94Wmh1ZvQ" target="_blank">arXiv:1707.04885</a>
        </p>
        <p class="text">
            <strong>180516</strong>: Our new preliminary working report is online: Multi-task Learning for Macromolecule Classification, Segmentation and Coarse Structural Recovery in Cryo-Tomography. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.06332&sa=D&sntz=1&usg=AFQjCNEJAdsHh1ZV1DtigT9hzbjzi-DKfw" target="_blank">arXiv:1805.06332</a>
        </p>
        <p class="text">
            <strong>180515</strong>: Our new preliminary working report is online: Image-derived generative modeling of pseudo-macromolecular structures - towards statistical assessment of electron cryotomography template matching. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.04634&sa=D&sntz=1&usg=AFQjCNGKj3NyL-PIBryMQi3zOw1_EJZ6fw" target="_blank">arXiv:1805.04634</a>
        </p>
        <p class="text">
            <strong>180504</strong>: Congratulations to Chang Liu and Xiangrui Zeng and other students for having following paper accepted: Deep learning based supervised semantic segmentation of Electron Cryo-Subtomograms. IEEE International Conference on Image Processing (ICIP) 2018. Preprint: <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1802.04087&sa=D&sntz=1&usg=AFQjCNE5BalTN63Z8cWmMj0qAIiUxCb7Qw" target="_blank">arXiv:1802.04087</a>
        </p>
        <p class="text">
            <strong>180402</strong>: Congratulations Yixiu Zhao and Xiangrui Zeng on their co-first authored paper being accepted by ISMB 2018! <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1804.01203&sa=D&sntz=1&usg=AFQjCNFUZrHiXv_yh0KWf1DO4qAy1IHOWQ" target="_blank">Title: An integration of fast alignment and maximum-likelihood methods for electron subtomogram averaging and classification</a>.
        </p>
        <p class="text">
            <strong>180321</strong>: Congratulations Jialiang Guo and Bo Zhou for their <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1801.10597&sa=D&sntz=1&usg=AFQjCNFQRg0GTEN8s2EaJSww7Dc-lCZHBg" target="_blank">paper</a> being accepted by ICIAR!
        </p>
        <p class="text">
            <strong>180212</strong>: Our new preliminary working report is online: <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1802.04087&sa=D&sntz=1&usg=AFQjCNE5BalTN63Z8cWmMj0qAIiUxCb7Qw" target="_blank">Deep learning based supervised semantic segmentation of Electron Cryo-Subtomograms</a>
        </p>
        <p class="text">
            <strong>180131</strong>: Our new preliminary working report is online: <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1801.10597&sa=D&sntz=1&usg=AFQjCNFQRg0GTEN8s2EaJSww7Dc-lCZHBg" target="_blank">Model compression for faster structural separation of macromolecules captured by Cellular Electron Cryo-Tomography</a>
        </p>
        <p class="text">
            <strong>171227</strong>: Congratulations Xiangrui Zeng for his first authored paper being accepted by Journal of Structural Biology: <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.jsb.2017.12.015&sa=D&sntz=1&usg=AFQjCNHd6jJBB-Obp_ebGPvYCwwXv9UrkA" target="_blank">A convolutional autoencoder approach for mining features in cellular electron cryo-tomograms and weakly supervised coarse segmentation</a>
        </p>
        <p class="text">
            <strong>170717</strong>: Our new preliminary working report is online: "<a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1707.04885&sa=D&sntz=1&usg=AFQjCNFVQ0NVTvzjbm5EVYnmj94Wmh1ZvQ" target="_blank">Improved deep learning based macromolecules structure classification from electron cryo tomograms</a>"
        </p>
        <p class="text">
            <strong>170616</strong>: Our new preliminary working report is online: "<a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1706.04970&sa=D&sntz=1&usg=AFQjCNFez1S5-gvM5whMCXe1yU5-vXVa2g" target="_blank">A convolutional autoencoder approach for mining features in cellular electron cryo-tomograms and weakly supervised coarse segmentation</a>"
        </p>
        <p class="text">
            <strong>170511</strong>: Kaiwen Wang just got awarded <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.cmu.edu%2Furo%2Fsura%2F&sa=D&sntz=1&usg=AFQjCNGWOFhvGMFlXiIkn11HXoBYpjog3g" target="_blank">CMU Summer Undergraduate Research Apprenticeship</a> on one of our collaborative research projects. Congratulations Kaiwen!
        </p>
        <p class="text">
            <strong>170406</strong>: Our manuscript entitled: "Deep learning based subdivision approach for large scale macromolecules structure recovery from electron cryo tomograms" will appear on ISMB 2017. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1701.08404&sa=D&sntz=1&usg=AFQjCNFIv5ZZ4mZCfToeLMsuslJpZ1s2Aw" target="_blank">Here</a> is a preprint version.
        </p>
        <p class="text">
            <strong>170129</strong>: Our new manuscript is online: "<a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1701.08404&sa=D&sntz=1&usg=AFQjCNFIv5ZZ4mZCfToeLMsuslJpZ1s2Aw" target="_blank">Deep learning based subdivision approach for large scale macromolecules structure recovery from electron cryo tomograms</a>"
        </p>
        <p class="text">
            <strong></strong>
        </p>
        

    </div>

    <div id="papers_page">
        <p class="title">Papers & Preprints</p>

        <p class="text"><strong>See <a href="https://scholar.google.com/citations?hl=en&user=Y3Cqt0cAAAAJ&view_op=list_works&sortby=pubdate" target="_blank">here</a> for the full list.</strong></p>

        <p class="text">
            0. Zeng X, Xu M. <b> Gum-Net: Unsupervised Geometric Matching for Fast and Accurate 3D  Subtomogram Image Alignment and Averaging </b>. In Proceedings of the IEEE conference on computer vision and pattern recognition (CVPR 2020).
        </p>

        <p class="text">
            1. Shi J, Zeng X, Jiang R, Jiang T, Xu M. A simulated annealing approach for resolution guided homogeneous cryo electron microscopy image selection. Quantitative Biology. doi:10.1007/s40484-019-0191-8. <a href="http://www.google.com/url?q=http%3A%2F%2Fjournal.hep.com.cn%2Fqb%2FEN%2F10.1007%2Fs40484-019-0191-8&sa=D&sntz=1&usg=AFQjCNEHVzCgwPUHt_M9Gow_dXWJZdJxjw" target="_blank">Paper</a>.
        </p>

        <p class="text">
            2. Zeng X, Xu M. <b> AITom: Open-source AI platform for cryo-electron Tomography data analysis </b>. (2019) <a href="https://arxiv.org/abs/1911.03044" target="_blank">arXiv:1911.03044</a>
        </p>
        <p class="text">
            3. Lu Y, Zeng X, Tian X, Shi X, Wang H, Zheng X, Liu X, Zhao X, Gao X, Xu M. Spark-based parallel calculation of 3D Fourier shell correlation for macromolecule structure local resolution estimation. Asia Pacific Bioinformatics Conference (APBC 2020).
        </p>
        <p class="text">
            4. Wu X, Zeng X, Zhu Z, Gao X, Xu M. (2019) Template-Based and Template-Free Approaches in Cellular Cryo-Electron Tomography Structural Pattern Mining. In: Computational Biology, (Husi H ed), Codon Publications,  <a href="https://www.google.com/url?q=https%3A%2F%2Fdx.doi.org%2F10.15586%2Fcomputationalbiology.2019.ch11&sa=D&sntz=1&usg=AFQjCNEXIuKWJRdyKJeULzHPDZdVJ5qQ1w" target="_blank">doi:10.15586/computationalbiology.2019.ch11</a>
        </p>
        <p class="text">
            5. Che C, Xian Z, Zeng X, Gao X, Xu M. Domain Randomization for Macromolecule Structure Classification and Segmentation in Electron Cyro-tomograms. IEEE International Conference on Bioinformatics and Biomedicine (BIBM 2019)
        </p>
        <p class="text">
            6. Wu X, Mao Y, Wang H, Zeng X, Gao X, Xing E, Xu M. Regularized Adversarial Training (RAT) for Robust Cellular Electron Cryo Tomograms Classification. IEEE International Conference on Bioinformatics and Biomedicine (BIBM 2019)
        </p>
        <p class="text">
            7. Du X, Zeng X, Zhou B, Singh A, Xu M. <b> Open-set Recognition of Unseen Macromolecules in Cellular Electron Cryo-Tomograms by Soft Large Margin Centralized Cosine Loss </b>. British Machine Vision Conference (BMVC 2019 spotlight with acceptance rate &#60; 5%). <a href="https://www.google.com/url?q=https%3A%2F%2Fbmvc2019.org%2Fwp-content%2Fuploads%2Fpapers%2F0347-paper.pdf&sa=D&sntz=1&usg=AFQjCNHsPGgSr-V9nd4XQgRo6XrbD-Gw3g" target="_blank">pdf</a>
        </p>
        <p class="text">
            8. Liu S, Du X, Xi R, Xu F, Zeng X, Zhou B, Xu M. Semi-supervised Macromolecule Structural Classification in Cellular Electron Cryo-Tomograms using 3D Autoencoding Classifier. British Machine Vision Conference (BMVC 2019). <a href="https://www.google.com/url?q=https%3A%2F%2Fbmvc2019.org%2Fwp-content%2Fuploads%2Fpapers%2F0337-paper.pdf&sa=D&sntz=1&usg=AFQjCNHC-ua4vqU6dKkVAI37462EuBwnhw" target="_blank">pdf</a>
        </p>
        <p class="text">
            9. Lu Y, Zeng X, Zhao X, Li S, Li H, Gao X, Xu M. Fine-grained Alignment of Cryo-electron Subtomograms Based on MPI Parallel Optimization. BMC Bioinformatics (2019) 20:443. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1186%2Fs12859-019-3003-2&sa=D&sntz=1&usg=AFQjCNHPqeNDeZFQ8DbqgD3r4fC2B6M_Eg" target="_blank">doi:10.1186/s12859-019-3003-2</a>
        </p>
        <p class="text">
            10. Lin R, Zeng X, Kitani K, Xu M. Adversarial domain adaptation for cross data source macromolecule in situ structural classification in cellular electron cryo-tomograms. Bioinformatics. 2019 Jul 5; 35(14); i260–i268. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1093%2Fbioinformatics%2Fbtz364&sa=D&sntz=1&usg=AFQjCNHxhJm3rCbHUvKF_b8lFMfKlOPxRQ" target="_blank">doi:10.1093/bioinformatics/btz364</a>
        </p>
        <p class="text">
            11. Han R, Bao Z, Zeng X, Niu T, Zhang F, Xu M, Gao X. A joint method for marker-free alignment of tilt series in electron tomography. Bioinformatics. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1093%2Fbioinformatics%2Fbtz323&sa=D&sntz=1&usg=AFQjCNEkKBLr2mRVz5nIQWjZOwrS8qQlFg" target="_blank">doi:10.1093/bioinformatics/btz323</a>
        </p>
        <p class="text">
            12. Wang W, Taft D, Chen Y, Zhang J, Wallace C, Xu M, Watkins S, Xing J. Learn to segment single cells with deep distance estimator and deep cell detector. Computers in Biology and Medicine. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.compbiomed.2019.04.006&sa=D&sntz=1&usg=AFQjCNF0Y1Lx7OAEalJ5fy5e-PGkbppwuQ" target="_blank">doi:10.1016/j.compbiomed.2019.04.006</a>
        </p>
        <p class="text">
            13. Luo Z, Zeng X, Bao Z, Xu M. Deep Learning-Based Strategy For Macromolecules Classification with Imbalanced Data from Cellular Electron Cryotomography. International Joint Conference on Neural Networks (IJCNN 2019). <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1908.09993&sa=D&sntz=1&usg=AFQjCNEGGz5bPVwV_mULYoK29LpmHoZcyA" target="_blank">arXiv:1908.09993</a>
        </p>
        <p class="text">
            14. Li R, Zeng X, Siegmund S, Lin R, Zhou B, Liu C, Wang K, Jiang R, Freyberg Z, Lv H, Xu M. Automatic Localization and Identification of Mitochondria in Cellular Electron Cryo-Tomography using Faster-RCNN. BMC Bioinformatics. 201920 (Suppl 3) :132 <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1186%2Fs12859-019-2650-7&sa=D&sntz=1&usg=AFQjCNHqvXvIEgkmSGKKbJRZ7YOB7ee_Gw" target="_blank">doi:10.1186/s12859-019-2650-7</a>
        </p>
        <p class="text">
            15. Gubins et al. Classification in Cryo-Electron Tomograms. Eurographics Workshop on 3D Object Retrieval. doi: 10.2312/3dor.20191061
        </p>
        <p class="text">
            16. Zhao G, Zhou B, Wang K, Jiang R, Xu M. <b> Respond-CAM: Analyzing Deep Models for 3D Imaging Data by Visualizations </b>. Medical Image Computing & Computer Assisted Intervention (MICCAI) 2018.   <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1806.00102&sa=D&sntz=1&usg=AFQjCNGgP4R3YTn18pWjf3tItO1EJcBocg" target="_blank">arXiv:1806.00102</a>
        </p>
        <p class="text">
            17. Liu C, Zeng X, Guo Q, Wang K, Xu M. Multi-task Learning for Macromolecule Classification, Segmentation and Coarse Structural Recovery in Cryo-Tomography. British Machine Vision Conference (BMVC) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.06332&sa=D&sntz=1&usg=AFQjCNEJAdsHh1ZV1DtigT9hzbjzi-DKfw" target="_blank">arXiv:1805.06332</a>
        </p>
        <p class="text">
            18. Wang K, Zeng X, Liang X, Huo Z, Xing E, Xu M. Image-derived generative modeling of pseudo-macromolecular structures - towards statistical assessment of electron cryotomography template matching. British Machine Vision Conference (BMVC) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.04634&sa=D&sntz=1&usg=AFQjCNGKj3NyL-PIBryMQi3zOw1_EJZ6fw" target="_blank">arXiv:1805.04634</a>
        </p>
        <p class="text">
            19. Liu C, Zeng X, Lin R, Liang X, Freyberg Z, Xing E, Xu M. Deep learning based supervised semantic segmentation of Electron Cryo-Subtomograms. IEEE International Conference on Image Processing (ICIP) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1802.04087&sa=D&sntz=1&usg=AFQjCNE5BalTN63Z8cWmMj0qAIiUxCb7Qw" target="_blank">arXiv:1802.04087</a>
        </p>
        <p class="text">
            20. Zhao Y, Zeng X, Guo Q, Xu M. <b> An integration of fast alignment and maximum-likelihood methods for electron subtomogram averaging and classification </b>. ISMB 2018. Bioinformatics. 2018 Jul 1; 34(13): i227–i236. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1093%2Fbioinformatics%2Fbty267&sa=D&sntz=1&usg=AFQjCNEzaiJD5hLMLjD4MIKW2O9gJ-Icvg" target="_blank">doi:10.1093/bioinformatics/bty267</a>. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1804.01203&sa=D&sntz=1&usg=AFQjCNFUZrHiXv_yh0KWf1DO4qAy1IHOWQ" target="_blank">arXiv:1804.01203</a>
        </p>
        <p class="text">
            21. Guo J, Zhou B, Zeng X, Freyberg Z, Xu M. Model compression for faster structural separation of macromolecules captured by Cellular Electron Cryo-Tomography.  <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1801.10597&sa=D&sntz=1&usg=AFQjCNFQRg0GTEN8s2EaJSww7Dc-lCZHBg" target="_blank">arXiv:1801.10597</a>. International Conference on Image Analysis and Recognition (ICIAR) 2018
        </p>
        <p class="text">
            22. Zhou B, Guo Q, Zeng X, Gao X, Xu M. Feature Decomposition Based Saliency Detection in Electron Cryo-Tomograms. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1801.10562&sa=D&sntz=1&usg=AFQjCNHaPv2_pv7s6zdRSGiM8levJDld8Q">arXiv:1801.10562</a>. IEEE International Conference on Bioinformatics & Biomedicine, Workshop on Machine Learning in High Resolution Microscopy (BIBM-MLHRM 2018)
        </p>
        <p class="text">
            23. Che C, Lin R, Zeng X, Elmaaroufi K, Galeotti J, Xu M. Improved deep learning based macromolecules structure classification from electron cryo tomograms. Machine Vision and Applications (in press). doi:10.1007/s00138-018-0949-4.  <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1707.04885&sa=D&sntz=1&usg=AFQjCNFVQ0NVTvzjbm5EVYnmj94Wmh1ZvQ" target="_blank">arXiv:1707.04885</a>
        </p>
        <p class="text">
            24. Zeng X, Leung M, Zeev-Ben-Mordehai T, Xu M. <b> A convolutional autoencoder approach for mining features in cellular electron cryo-tomograms and weakly supervised coarse segmentation </b>. Journal of Structural Biology.  2018 May;202(2):150-160. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.jsb.2017.12.015&sa=D&sntz=1&usg=AFQjCNHd6jJBB-Obp_ebGPvYCwwXv9UrkA" target="_blank">doi:10.1016/j.jsb.2017.12.015</a>  <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1706.04970&sa=D&sntz=1&usg=AFQjCNFez1S5-gvM5whMCXe1yU5-vXVa2g" target="_blank">arXiv:1706.04970</a> [<a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fxulabs%2Fprojects%2Ftree%2Fmaster%2Fautoencoder&sa=D&sntz=1&usg=AFQjCNG3kvV1s4UAW8nIBaqZN67rbqN-uA">code</a>]
        </p>
        <p class="text">
            25. Xu M, Chai X, Muthakana H, Liang X, Yang G, Zeev-Ben-Mordehai T, Xing E. Deep learning based subdivision approach for large scale macromolecules structure recovery from electron cryo tomograms. 2017. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1701.08404&sa=D&sntz=1&usg=AFQjCNFIv5ZZ4mZCfToeLMsuslJpZ1s2Aw" target="_blank">arXiv:1701.08404</a>.  ISMB 2017 (acceptance rate 16%), Bioinformatics <a href="http://www.google.com/url?q=http%3A%2F%2Fdx.doi.org%2F10.1093%2Fbioinformatics%2Fbtx230&sa=D&sntz=1&usg=AFQjCNGmko1GllIVm8-m-l30YjSPRVFS9w" target="_blank">doi:10.1093/bioinformatics/btx230</a> [<a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fxulabs%2Fprojects%2Fblob%2Fmaster%2Fparticle_subdivision%2Fsubdivide.py&sa=D&sntz=1&usg=AFQjCNGBfTfETvntmRlmsGiFvDXb8q_EgA" target="_blank">code</a>]
        </p>
        <p class="text">
            26. Xu M, Singla J, Tocheva E, Chang Y, Stevens R, Jensen G, Alber F. <b> De novo structural pattern mining in cellular electron cryo-tomograms </b>. Structure. 2019 Apr 2;27(4):679-691.e14.  <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.str.2019.01.005&sa=D&sntz=1&usg=AFQjCNG4CwvStj7Z7IutfOKq_h8TcKdxZw" target="_blank">doi:10.1016/j.str.2019.01.005</a>.  (Appeared on Structure volume cover and highlighted in Nature Methods 16, page 285 (2019), <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1038%2Fs41592-019-0382-2&sa=D&sntz=1&usg=AFQjCNHMCQyejm_T4RGn8t5EKPadA4XFzA" target="_blank">doi:10.1038/s41592-019-0382-2</a>)
        </p>
        <p class="text">
            27. Frazier Z, Xu M, Alber F. TomoMiner and TomoMiner Cloud: A software platform for large-scale subtomogram structural analysis. Structure 2017. (doi: 10.1016/j.str.2017.04.016, co-corresponding author).
        </p>
        <p class="text">
            28. Pei L, Xu M, Frazier Z, Alber F. Simulating Cryo-Electron Tomograms of Crowded Mixtures of Macromolecular Complexes and Assessment of Particle Picking. BMC Bioinformatics. 2016; 17: 405.
        </p>
        <p class="text">
            29. Xu M, Alber F. Automated target segmentation and real space fast alignment methods for high-throughput classification and averaging of crowded cryo-electron subtomograms. ISMB/ECCB 2013, Bioinformatics. 2013 Jul 1;29(13):i274-82.
        </p>
        <p class="text">
            30. Thalassinos K, Pandurangan AP, Xu M, Alber F, Topf M. Conformational States of macromolecular assemblies explored by integrative structure calculation. Structure. 2013 Sep 3;21(9):1500-8.
        </p>
        <p class="text">
            31. Xu M, Beck M, Alber F. <b> High-throughput subtomogram alignment and classification by Fourier space constrained fast volumetric matching </b>. Journal of Structural Biology. 2012 May;178(2):152-64. Epub 2012 Mar 7.
        </p>
        <p class="text">
            32. Xu M, Alber F. High precision alignment of cryo-electron subtomograms through gradient-based parallel optimization. BMC Systems Biology. 2012, 6(Suppl 1):S18
        </p>
        <p class="text">
            33. Xu M, Beck M, Alber F. Template-free detection of macromolecular complexes in cryo-electron tomograms. Bioinformatics (ISMB 2011). 2011 Jul 1;27(13):i69-i76.
        </p>
        <p class="text">
            34. Beck M, Topf M, Frazier Z, Tjong H, Xu M, Zhang S, Alber F. Exploring the Spatial and Temporal Organization of a Cell’s Proteome. Journal of Structural Biology. 2011 Mar; 173(3):483-496.
        </p>
        <p class="text">
            35. Zhang S, Vasishtan D, Xu M, Topf M, Alber F. A fast mathematical programming procedure for simultaneous fitting of assembly components into cryo-EM density maps. Bioinformatics (ISMB 2010). 2010 Jun 15;26(12):i261-8.
        </p>
        <p class="text">
            36. Xu M, Zhang S, Alber F. 3D rotation invariant features for characterization of molecular density map images. IEEE International Conference on Bioinformatics & Biomedicine (BIBM 2009)
        </p>


    </div>

    <div id="group_page">
        <p class="title">Group</p>

        <p class="topic"><strong>PhD students</strong></p>
        <img class="image_group", src="https://user-images.githubusercontent.com/25089434/73482248-5d0e4780-436b-11ea-92c2-16ab85109c37.jpg">
        <p class="text"><a href="https://scholar.google.com/citations?hl=en&user=8gQLySoAAAAJ&view_op=list_works&sortby=pubdate" target="_blank">Xiangrui Zeng</a></p>

        <p class="topic"><strong>Research assistants</strong></p>
        <p class="text">
            Kaiwen Wang (ECE) <br />
            Yixiu Zhao (CSD) <br />
            Bo Zhou (RI, MSCV) <br />
            Chang Liu (ECE) <br />
        </p>

        <p class="topic"><strong>Students in collaboration</strong></p>
        <p class="text">
            (Please see our publication)
        </p>

    </div>

    <div id="aboutxu_page">
        <p class="title">About Min Xu</p>


        <img class="image" src="https://user-images.githubusercontent.com/25089434/73482242-597ac080-436b-11ea-84f6-6321240648bf.png">
        <p style="text-align: center;">
            <strong>Min Xu, Ph.D.</strong><br />
            Assistant Research Professor, <a href="http://www.google.com/url?q=http%3A%2F%2Fcbd.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNH2a-HoqIwz5FlmV2iqBe6MQqLAZg" target="_blank">Computational Biology Department</a><br />
            Training faculty, <a href="http://www.google.com/url?q=http%3A%2F%2Fri.cmu.edu%2Feducation%2Facademic-programs%2Fmaster-of-science-computer-vision-mscv&sa=D&sntz=1&usg=AFQjCNF6MjntgO_YKppTAS6wa1-_1z8clg" target="_blank">Master of Science in Computer Vision</a>, <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.ri.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNGcZ87enxg9p6syPq-tQXu0D1_Tcg" target="_blank">Robotics Institute</a><br />
            <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.cs.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNHOBZa6FUtlVkCoqtax6zX1O2G-UQ" target="_blank">School of Computer Science</a>
            <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNGgVsrmaGAPErZd_9YZXZtyZanpvA" target="_blank">Carnegie Mellon University</a><br />
            <br />

            <strong>Office: </strong>
            GHC 7709, 5000 Forbes Avenue, Pittsburgh, PA 15213<br />
            <strong>Email: </strong>mxu1 @ cs.cmu.edu<br />
            <br />

            <strong>
            <a href="https://docs.google.com/document/d/e/2PACX-1vQOw1_7_1a6HuyyvcwmtOzfliOmyz4L7giYkp3-i0MOxh-ASU0ZCjNtdYZZ4KC3UlFHdC9Gb-gzXMsi/pub" target="_blank">Curriculum Vitae</a>,
            <a href="http://scholar.google.com/citations?hl=en&user=Y3Cqt0cAAAAJ&view_op=list_works&sortby=pubdate" target="_blank">Google Scholar</a>, <a href="https://www.google.com/url?q=https%3A%2F%2Fwww.researchgate.net%2Fprofile%2FMin_Xu21&sa=D&sntz=1&usg=AFQjCNGyDs_50pSYbPIdxXWUW1PxHBfd6g" target="_blank">Research Gate</a>, and <a href="https://www.linkedin.com/in/min-xu-45b7315" target="_blank">Linkedin</a>  pages.<br />
            </strong>
        </p>

        
        <p class="text">
            <p class="topic"> <strong>Biography</strong></p>
            <p>
                Dr. Min Xu is an Assistant Research Professor at the <a href="http://www.google.com/url?q=http%3A%2F%2Fcbd.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNH2a-HoqIwz5FlmV2iqBe6MQqLAZg" target="_blank">Computational Biology Department</a> in the <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.cs.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNHOBZa6FUtlVkCoqtax6zX1O2G-UQ" target="_blank">School of Computer Science</a> at <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNGgVsrmaGAPErZd_9YZXZtyZanpvA" target="_blank">Carnegie Mellon University</a>. 
                He serves as training faculty at the <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.compbio.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNFADTqAWyeNIUIugdw0eYUjxX2otA" target="_blank">Joint CMU-Pitt Ph.D. Program in Computational Biology</a>. 
                He also serves as a training faculty <a href="http://www.google.com/url?q=http%3A%2F%2Fri.cmu.edu%2Feducation%2Facademic-programs%2Fmaster-of-science-computer-vision-mscv&sa=D&sntz=1&usg=AFQjCNF6MjntgO_YKppTAS6wa1-_1z8clg" target="_blank">Master of Science in Computer Vision</a> Program at <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.ri.cmu.edu%2F&sa=D&sntz=1&usg=AFQjCNGcZ87enxg9p6syPq-tQXu0D1_Tcg" target="_blank">Robotics Institute</a>. 
                He is an investigator at the National Center for Multiscale Modeling of Biological Systems.
            </p>
            <p>
                Dr. Xu’s career has centered on developing computational methods for the study of cellular systems using imaging and omics data. 
                He started his research career in the field of Computational Biology and Bioinformatics since 2000. 
                He developed machine learning methods for gene selection for classifying cancer samples, for cancer gene network module discovery, and for sample phenotype prediction through integration of hundreds of gene expression datasets. 
                Since 2008, he started working in the field of computational analysis of Cellular Electron CryoTomography (CECT) data. 
                He designed structural pattern mining methods and first demonstrated the feasibility of De Novo extraction of structures and spatial organizations of macromolecular complexes in single cells using CECT data. 
                His current research focus on CECT derived modelling of cellular organization at molecular resolution.
            </p>
            <p>
                Dr. Xu has published over 40 research papers. He is currently serving in the editorial board of Statistical Methods in Medical Research.<br />
                Dr. Xu received an B.E. in Computer Science from the Beihang University, M.Sc from School of Computing at the National University of Singapore, M.A. in Applied Mathematics from the University of Southern California (USC), and Ph.D. in Computational Biology and Bioinformatics from USC. 
                He was a postdoctoral researcher at USC.
            </p>
            
            <p>
                <strong>Dr. Xu does research for fun, and he enjoys working with highly talented and motivated students.</strong>
            </p>
            
        </p>
    </div  >

    <div id="position_page">
        <p class="title">Positions</p>

        <p class="text">
            We are looking for PhD students to join our lab!<br />
            We have funded research assistant positions available!<br />
            We are looking for motivated students to join our lab for graduate or undergraduate level collaborative research projects!<br />
            We welcome international research interns!<br />
            We also welcome remote collaborations!<br />
            <br />
            Interested please email Dr. Xu for further information.<br />
        </p>
    </div>

    <div id="software_page">
        <p class="title">Software</p>

        <p class="text">
            The source code associated with our individual papers can be found at our papers page, following with each paper. 
            All source code are deposited to <a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fxulabs&sa=D&sntz=1&usg=AFQjCNGKgTx2tv6az6lc_YAdRBGHA5DQAQ" target="_blank">our github</a> portal. We are also in the process of integrating the algorithms into our <a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fxulabs%2Faitom&sa=D&sntz=1&usg=AFQjCNGHL8KCuMCGX75G0AZx4R4hV0264Q" target="_blank">AITom</a> platform. 
        </p>
    </div>
</div>


<div class="home_page" style="border: none;">

        <p class="title">Research</p>
	
    
        <p class=“text”>The <a href="https://www.google.com/url?q=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FCell_%28biology%29&sa=D&sntz=1&usg=AFQjCNF_7gaTbtFsB0Ki6GeCnXp6lBYIqA" target="_blank">cell</a> is the basic structural, functional, and biological unit of all known <a href="https://www.google.com/url?q=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FLife&sa=D&sntz=1&usg=AFQjCNG9DyiXnxyksf0_G5RS_SjanpI7kw" target="_blank">living</a> organisms. 
            It is a tiny but very complicated "living machine" that can do a lot of amazing things. 
            However, so far we have very limited knowledge about its complicated molecular machinery 
            due to lack of high resolution and systems level data of individual cells. 
            Cellular <a href="https://www.google.com/url?q=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FCryo-electron_tomography&sa=D&sntz=1&usg=AFQjCNHWEseuCR7YEmlHHCVWMhC3eGdVzA" target="_blank">Electron CryoTomography</a> is an emerging imaging technique that captures the 3D electron density distribution of cells 
            at sub-molecular resolution and at close-to-native state. Our lab aims to use cutting-edge computation, 
            mathematics and <a href="https://www.google.com/url?q=https%3A%2F%2Fai.cs.cmu.edu&sa=D&sntz=1&usg=AFQjCNETDTm2QNYrV4pogAMBg7kIzhvTjg" target="_blank">artificial intelligence</a> techniques, particularly those related to computer vision, machine learning and big data, 
            to build structural organization models using such imaging data. 
            Such modeling would be useful for getting new insights into the machinery of cellular systems.
        </p>
    
        <img class="image", src="https://user-images.githubusercontent.com/25089434/73482186-3fd97900-436b-11ea-8ee4-6599ac0e0da3.png">
    
        <p class="text">
            The figure above shows extracted structural patterns in cellular tomograms: 
            (a) Slices of 3D tomogram images of two bacterial cells. Image data are from the <a href="http://www.google.com/url?q=http%3A%2F%2Fjensenlab.caltech.edu%2F&sa=D&sntz=1&usg=AFQjCNFSimic2PAB0tDQxCDlvQEQ58A33A" target="_blank">Jensen Lab</a> at Caltech. 
            (b) Isosurfaces of instances of extracted structural patterns embedded into the original images. 
            (c) Embedded instances, zooming in on a particular region. 
            (d) Isosurfaces of one example structural pattern from each experiment. 
            (e) Spatial distributions of instances of different structural patterns: 
            left: the Ribosome like patterns distributed outside the nucleoid region; 
            middle: patterns distributed on the nucleoid region; right: patterns distributed at the tip of the cell. 
            For details, see <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.str.2019.01.005&sa=D&sntz=1&usg=AFQjCNG4CwvStj7Z7IutfOKq_h8TcKdxZw" target="_blank">our paper</a> on Structure.
        </p>
</div>


<!-- jQuery -->
<script src="js/jquery.js"></script>
<!-- nav -->
<script src="js/jquery.scrollTo.js"></script>
<!-- localScroll -->
<script src="js/jquery.localscroll-1.2.7-min.js"></script>
<!-- bootstrap -->
<script src="js/bootstrap.js"></script>
<!-- switch pages -->
<script src='js/jquery.min.js'></script>
</body>
</html>
